### PR TITLE
Initial commit: setup project structure 

### PR DESCRIPTION
**Pull request by AlrauneZ**
Setting up the project with directories and add on files (config, readme etc.) following the NL EScience center python project template.

### CODE REVIEW

**Review by raar1 — APPROVED** [review]
Good work setting up the project structure!

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# YAML 1.2
+---
+cff-version: "1.2.0"
+title: "mibipret"
+authors:
+  -
+    family-names: Zech
+    given-names: Alraune
+    orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (JaroCamphuijsen) [inline CITATION.cff:9]: Maybe you can put your ORCID here. That will make it easier for people to find you.

Comment (JaroCamphuijsen) [reply CITATION.cff:9]:
```suggestion
    orcid: "https://orcid.org/0000-0002-8783-6198"
```

**Review by sonarima — APPROVED** [review]
thanks, looks fine!

**Review by JaroCamphuijsen — APPROVED** [review]

**Pull request merged**